packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -80,6 +80,15 @@ If false the default placeholder style is used.
 - Type: `Boolean`
 - Required: No
 - Default: `false`
+- Platform: Web | Mobile
+
+### isSelected
+
+Whether the block is selected or not.
+
+- Type: `Boolean`
+- Required: No
+- Platform: Mobile
```

Comment:
It doesn't look like `isSelected` needs to be part of MediaPlaceholder API. Block can decide to render/not render the MediaPlaceholder due to its selected state. 
Also as far as I see web also handles it this way without needing to add this to MediaPlaceholder API and we want to come close to web as much as possible.
Let me know if I am missing something.

Reply:
According to the mobile gallery [design](https://app.zeplin.io/project/5bfecbea71ce881122eadd2f/screen/5d3b1273384a876ce6c195b3), `isAppender` button should be displayed only when `gallery` block is selected, that's why there is a need to pass `isSelected` value from parent component into `mediaPlaceholder`. Please, correct me if I understood it wrongly.

Reply:
Could you be able to do some investigation about how this works on web? As far as I see mediaplaceholder isn't rendered if block is not selected on web as well. So how is it handled without some isSelected prop? We want to keep props of components as similar as possible between platforms. 
Independent from this, it doesn't sound right for a component to have a prop named 'isSelected' where it actually decides rendering/not rendering it.

Reply:
Sure, will investigate it!

Reply:
After the investigation, I've noticed that web media-placeholder accepts prop called `dropZoneUIOnly={ hasImages && ! isSelected }` which then within placeholder is rendering gallery with appender when `dropZoneUIOnly === false` otherwise without appender.

Reply:
Maybe instead of adding new prop `isSelected` I should reuse and document prop `dropZoneUIOnly`, wdyt?

Reply:
maybe we can make use of the same prop for parity, although, we don't currently have drag & drop of media. But it looks like a nice enhancement for the future. wdyt @koke ? I think we can start using `dropZoneUIOnly` on mobile too.

Reply:
I don't think `dropZoneUIOnly` is the right prop to use here. I checked the `README` and it's not documented anywhere. From what I'm seeing, as long as media upload is possible, the drop zone would be rendered on the web, and the UI to pick media is the variable here.

Adding `dropZoneUIOnly` to mobile would imply that we support drag and drop, and could be confusing. I'd rather flip the logic and have a `showMediaSelectionUI` (or another better name) prop that is set only when the block is selected.

Reply:
The name you've proposed (`showMediaSelectionUI`) sounds good for me. Just to be sure, would you like to replace `dropZoneUIOnly` with `showMediaSelectionUI` on the web and have the same prop on mobile?

Reply:
Yes, I would propose that change for the web as well

Reply:
LGTM as soon as we keep the parity. 👍